assets/js/Containers/FrevaGPT/index.js

```diff
@@ -233,42 +244,61 @@ class FrevaGPT extends React.Component {
         <Row>
           <div className="d-flex justify-content-between">
             <h2 onClick={this.toggleBotSelect}>FrevaGPT</h2>
+
+            <div className="d-flex justify-content-between mb-2">
+              <Form.Select 
+                value={this.botModel}
+                onChange={(x) => { this.setState({ botModel: x.target.value }); }}
+                className="me-1"
+                placeholder="Model"
+                hidden={this.state.hideBotModelList}>
+                {this.state.botModelList.map((x) => {
+                    return <option key={x}>{x}</option>;
+                })}
+              </Form.Select>
+              <Button onClick={this.createNewChat} variant="info">NewChat</Button>
+            </div>
           </div>
   
           <Col md={4}>
-            <Form.Select 
-              value={this.botModel}
-              onChange={(x) => { this.setState({ botModel: x.target.value }); }}
-              className="me-1 mb-3"
-              placeholder="Choose Chatbot"
-              hidden={this.state.hideBotModelList}>
-              {this.state.botModelList.map((x) => {
-                  return <option key={x}>{x}</option>;
-              })}
-            </Form.Select>
             <SidePanel/>
           </Col>
   
           <Col md={8}>
-
+            {this.state.showSuggestions ? (
```

Comment:
* Adding the buttons with flex and `m-2` will result in some annoying margins left and right of the buttons (the buttons are not fully aligned with the input-field) and they are not very mobile-friendly. Consider a solution based on Cols and Rows along with gutters:

```javascript
            {this.state.showSuggestions ? (
              <Row className="mb-2 g-2">
                {botSuggestions.map((element, index) => {
                  return (
                    <div key={index} className="col-md-3">
                      <Button
                        className="h-100 w-100"
                        variant="outline-secondary"
                        onClick={() => this.handleSubmit(element)}
                        value={element}
                      >
                        {truncate(element)}
                      </Button>
                    </div>
                  );
                })}
              </Row>
            ) : null}
```

* Since `element` seems always to be a string replace `key={index}` by `key={element}` to get rid of those pesky linter-warnings about the usage of `index`
* Consider adding a Tooltip to the Button with the full text so that the user knows what they are doing.

Reply:
Adapted accordingly: https://github.com/FREVA-CLINT/freva-web/pull/60/commits/89e616c6a968e8bbad678195a320c9caab1264be